package.json

```diff
@@ -5,6 +5,12 @@
 	"main": "./dist/lib/proto.js",
 	"types": "./dist/lib/proto.d.ts",
 	"license": "GPL-3.0",
+	"files": [
+		"dist"
```

Comment:
We need `config` folder as well - it includes contract ABI files needed in runtime (although folder name is crappy)

Reply:
Added in 2556cda commit